lib/WP_Auth0_Api_Operations.php

```diff
@@ -48,10 +48,10 @@ public function update_wordpress_connection( $app_token, $connection_id, $passwo
 		}
 
 		$login_script = str_replace( '{THE_WS_TOKEN}', $migration_token, WP_Auth0_CustomDBLib::$login_script );
-		$login_script = str_replace( '{THE_WS_URL}', get_home_url() . '/index.php?a0_action=migration-ws-login', $login_script );
+		$login_script = str_replace( '{THE_WS_URL}', site_url( 'index.php?a0_action=migration-ws-login' ), $login_script );
 
 		$get_user_script = str_replace( '{THE_WS_TOKEN}', $migration_token, WP_Auth0_CustomDBLib::$get_user_script );
-		$get_user_script = str_replace( '{THE_WS_URL}', get_home_url() . '/index.php?a0_action=migration-ws-get-user', $get_user_script );
+		$get_user_script = str_replace( '{THE_WS_URL}', site_url(), $get_user_script );
```

Comment:
why is this one `$get_user_script` but the previous one is  `index.php?a0_action=migration-ws-login` even though appears to be a `$login_script` ?

Reply:
@cocojoe - Eeks, big miss on my part. Fixing.